osf/management/commands/add_egap_admin.py

```diff
@@ -0,0 +1,38 @@
+# -*- coding: utf-8 -*-
+import logging
+
+from django.core.management.base import BaseCommand
+from django.contrib.auth.models import Group
+from waffle.models import Flag
+from osf import features
+from osf.models import OSFUser
+
+logger = logging.getLogger(__name__)
+
+
+def create_egap_admins_group(username):
+    # `get_or_create` used here so command can be reused for new admins.
+    flag, _ = Flag.objects.get_or_create(name=features.EGAP_ADMINS)
+    group, _ = Group.objects.get_or_create(name=features.EGAP_ADMINS)  # Just using the same name for convenience
```

Comment:
I don't think this is necessary. They can just add Matt to the flag in the admin admin. The only thing we really need is to create the flag, since [the management command for handling waffle flags](https://openscience.atlassian.net/browse/ENG-1410) isn't done yet.